src/Java.Interop.Tools.JavaCallableWrappers/Java.Interop.Tools.JavaCallableWrappers/CRC64.cs

```diff
@@ -0,0 +1,156 @@
+using System;
+using System.Security.Cryptography;
+
+namespace Java.Interop.Tools.JavaCallableWrappers
+{
+	/// <summary>
+	///  CRC64 variant: crc-64-jones 64-bit
+	///  * Poly: 0xad93d23594c935a9
+	/// </summary>
+	public class CRC64 : HashAlgorithm
+	{
+		static readonly ulong [] Table = {
```

Comment:
Where did this table & algorithm come from?  What's the license?

Reply:
Came from: https://github.com/gityf/crc/blob/master/crc/crc64.c

Then ported to C#, do I need to include their header comment in this file? Or can we even use it?

Reply:
It's a BSD 3-clause license, which should be fine.  However, we'll need to ensure that the license information makes its way into `xamarin-android/ThirdPartyNotices.txt`, and this file should probably also contain that license text.